### ISSUE TITLE
Dash and Kujira support?

### ISSUE THREAD

**Issue by GiMa-Maya** [shapeshift/hdwallet#632]
### Overview

Hey. El Dorado is working on integrating this SDK. Does this SDK support Dash and the Kujira blockchain? I see some Dash references. For Kujira I’m not sure. Kujira is a cosmos chain.

### References and additional details

-

### Acceptance Criteria

-

**Comment by BitHighlander** [comment]
yes on dash, full list here, https://github.com/shapeshift/hdwallet/blob/master/packages/hdwallet-core/src/utils.ts#L129

never herd of Kujira before

**Comment by GiMa-Maya** [comment]
Awesome, thank you for the list. Kujira is a cosmos chain that Maya Protocol will integrate this week. I can tell you more about it in the discord ticket we have open in the Keepkey server (recognise your name from that). Is it hard to integrate Kujira into this SDK (taking into account the Kujira is a cosmos chain...). Happy to do a PR.
